.github/dependabot.yaml

```diff
@@ -43,21 +43,12 @@ updates:
     # Keep Logback version 1.2.x
     - dependency-name: "ch.qos.logback:*"
       update-types: ["version-update:semver-major", "version-update:semver-minor"]
-    # Mockito 5.x requires Java 11
-    - dependency-name: "org.mockito:*"
-      update-types: ["version-update:semver-major"]
-    # JUnit Pioneer 2.x requires Java 11
-    - dependency-name: "org.junit-pioneer:*"
-      update-types: ["version-update:semver-major"]
```

Comment:
Are you sure, we can upgrade these?

These are test dependencies and tests are still compiled with `-release 8` and **must** run on JRE 8.

Reply:
:+1: Reverted.